release/__init__.py

```diff
@@ -429,6 +431,11 @@ def make_abs(path):
 
 
 def do_build_docker(name, path):
+    with logger.scope("dcos/dcos-builder"):
```

Comment:
can we include name here? For some versions of DC/OS we commonly build multiple dockers with this code.